src/connectedk8s/azext_connectedk8s/troubleshoot_diagnoser_job_without_proxycert.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: azure-arc-troubleshoot-sa
+  namespace: azure-arc
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: secret-reader
+  namespace: default
+rules:
+- apiGroups:
+  - ''
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: read-secrets-rolebindings
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: secret-reader
+subjects:
+- kind: ServiceAccount
+  name: azure-arc-troubleshoot-sa
+  namespace: azure-arc
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: networktest
```

Comment:
we discussed that we'll rename this right? since it's not networktest alone but has other check too. Something like arc diagnoser?